packages/orbit-design-tokens/src/dictionary/definitions/component/form-element/form-element-font-size.json

```diff
@@ -0,0 +1,54 @@
+{
+  "component": {
+    "form-element": {
+      "small": {
+        "font-size": {
+          "type": "size",
+          "value": "{foundation.font-size.normal}"
+        }
+      },
+      "normal": {
+        "font-size": {
+          "type": "size",
+          "value": "{foundation.font-size.normal}"
+        }
+      }
+    },
+    "font-size": {
+      "input": {
+        "small": {
+          "type": "size",
+          "value": "{foundation.font-size.normal}",
+          "deprecated": true,
+          "deprecated-replace": "{component.form-element.small.font-size}",
+          "deprecated-version": "1.0.0"
+        },
+        "normal": {
+          "type": "size",
+          "value": "{foundation.font-size.normal}",
+          "deprecated": true,
+          "deprecated-replace": "{component.form-element.normal.font-size}",
+          "deprecated-version": "1.0.0"
+        }
+      },
+      "form-feedback": {
```

Comment:
On the context of this question, why is this declared [here](https://skypicker.slack.com/archives/C04HMMQ6VNE/p1688480433368959)? Does it make sense to deprecate a token that only existed in the context of another repo? :thinking:
What is the value we are trying to achieve with this? Ease their migration? If so, I can understand it, but I am afraid it can leave to more dangling classes through time... WDYT?

Reply:
wdym in the context of only another repo?  

what's [this](https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/src/Checkbox/index.tsx#L122) and [this](https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/src/Checkbox/index.tsx#L122) then? :) 

> What is the value we are trying to achieve with this? Ease their migration? If so, I can understand it, but I am afraid it can leave to more dangling classes through time

In general, having one repo using deprecated tokens -> yes, that is supposed to help them to migrate, instead of making breaking changes immediately, we can properly communicate that and deprecate them first and have it as a part of the documentation.